addons/common/fnc_switchPlayer.sqf

```diff
@@ -43,13 +43,13 @@ _dummyUnit = (_ar select 4) createUnit [_type, [0, 0, 0], [], 0, "NONE"]; // Joi
 if (isNull _dummyUnit) exitWith { hint "Sorry, something went wrong, dummyUnit is null" };
 [_oldUnit] join _dummyGroup;
 
-LOG(format["1.Dummy created, State saved and put oldUnit in new group: %1", _dummyGroup]);
+// LOG(format["1.Dummy created, State saved and put oldUnit in new group: %1", _dummyGroup]);
```

Comment:
```
error: Preprocessor error: Function call with incorrect number of arguments, expected `1` got `3`. `Token { symbol: Word("LOG"), source: Position { start: (1551, (46, 1)), end: (1554, (46, 4)), path: "/addons/common/fnc_switchPlayer.sqf" }, parent: None }`

   --> addons/common/fnc_switchPlayer.sqf:46:1
    |
 46 | LOG(format["1.Dummy created, State saved and put oldUnit in new group: %1", _dummyGroup]);
    | ^ Expects 1 arguments
```
It hard stops on this error, I'd prefer it just throw a warning and skip this file

Reply:
This should be using `LOG_1` anyways.